pom.xml

```diff
@@ -147,21 +147,21 @@
       <!-- Spring boot version overrides - END -->
 
       <!-- Vaadin versions - START -->
-      <vaadin.spring.version>1.2.0</vaadin.spring.version>
-      <vaadin.spring.addon.version>0.0.6.RELEASE</vaadin.spring.addon.version>
-      <vaadin.version>7.7.13</vaadin.version>
+      <vaadin.version>8.11.1</vaadin.version>
+      <vaadin.spring.version>3.2.1</vaadin.spring.version>
+      <vaadin.spring.addon.version>2.0.0.RELEASE</vaadin.spring.addon.version>
       <vaadin.plugin.version>${vaadin.version}</vaadin.plugin.version>
-      <vaadin.addon.vaadin-lazyquerycontainer.version>7.6.1.3</vaadin.addon.vaadin-lazyquerycontainer.version>
-      <vaadin.addon.flexibleoptiongroup.version>2.2.0</vaadin.addon.flexibleoptiongroup.version>      
-      <vaadin.addon.dbar-addon.version>2.0.0</vaadin.addon.dbar-addon.version>
+      <vaadin.addon.dbar-addon.version>3.0.1</vaadin.addon.dbar-addon.version>
+      <vaadin.gwt-api.version>1.3.0</vaadin.gwt-api.version>
+      <vaadin.gwt-user.version>2.8.2</vaadin.gwt-user.version>
       <!-- Vaadin versions - END -->
 
       <maven.processor.plugin.version>3.3.3</maven.processor.plugin.version>
       <maven.scm.plugin.version>1.11.1</maven.scm.plugin.version>
 
       <!-- Misc libraries versions - START -->
       <cron-utils.version>5.0.5</cron-utils.version>
-      <jsoup.version>1.8.3</jsoup.version>
+      <jsoup.version>1.11.2</jsoup.version>
```

Comment:
Do we still require `jsoup`? At least it's a transitive dependency of `vaadin-server` in exactly this version

Reply:
soup is being used by the _hawkbit-repository-jpa/pom.xml_ as well. The question is should we leave it in version 1.8.3 as before and override it in _hawkbit-ui/pom.xml_ or update to 1.11.2 as currently configured?

Reply:
Well spotted! It seems like `jsoup` is an optional dependency of the `hibernate-validator` which is required in case you use some specific validators:

```
java.lang.NoClassDefFoundError: org/jsoup/safety/Whitelist
```